format/spec.md

```diff
@@ -1286,6 +1298,7 @@ This serialization scheme is for storing single values as individual binary valu
 | **`struct`**                 | Not supported                                                                                                |
 | **`list`**                   | Not supported                                                                                                |
 | **`map`**                    | Not supported                                                                                                |
+| **`geometry`**       | Always a single point, it is encoded in big-endian fashion as concatenated {x, y, optional z, optional m} values. |
```

Comment:
Why big endian? Most of the time we use little endian in the format, with the only exception being the encoded decimal values.

Reply:
Also, what is the encoding for these values? 8-byte IEEE 754?

Reply:
Makes sense, added little-endian and 8-byte IEEE 754 (ie double type) for each coordinate